workspaces/marketplace/plugins/marketplace/src/components/MarketplaceCatalogContent.tsx

```diff
@@ -134,6 +139,24 @@ export const MarketplaceCatalogContent = () => {
   const showExtensionsConfigurationAlert =
     !isProductionEnvironment && !extensionsConfig.data?.enabled;
 
+  const installedPluginsCount = Object.entries(installedPlugins)?.length ?? 0;
+  const pluginInfo = () => {
+    if (installedPluginsCount > 1) {
+      return (
+        <>
+          You have {installedPluginsCount} plugins that require a restart of
+          your backend system to either finish installing or updating.
+        </>
+      );
+    }
+    return (
+      <>
+        The <b>{Object.keys(installedPlugins)[0]}</b> plugin requires a restart
+        of the backend system to finish installing or updating.
+      </>
+    );
+  };
```

Comment:
Suggested this change to help separate UI text from logic, which will make it easier to extract strings when we add i18n support later.
```suggestion
const getPluginAlertMessage = () => {
  const pluginNames = Object.keys(installedPlugins);

  if (installedPluginsCount > 1) {
    return `You have ${installedPluginsCount} plugins that require a restart of your backend system to either finish installing or updating.`;
  }

  return `The ${pluginNames[0]} plugin requires a restart of the backend system to finish installing or updating.`;
};

const pluginInfo = () => {
  return <>{getPluginAlertMessage()}</>;
};
```

Reply:
When a single plugin is installed, the plugin name has to be in bold. If you notice, my alert here has the name in bold

```
<>
   The <b>{Object.keys(installedPlugins)[0]}</b> plugin requires a restart
    of the backend system to finish installing or updating.
</>
```